specifications/FiniteMonotonic/MCCRDT.tla

```diff
@@ -0,0 +1,67 @@
+------ MODULE MCCRDT -----
+EXTENDS CRDT, IOUtils, TLC
+
+Divergence == 
+    \* D=0 means there is no divergence. D=1 means Gossip synchronizes the system in a single step. 
+    atoi(IOEnv.D)
+ASSUME Divergence \in Nat
+
+Constraint ==
+    IOEnv.C = ToString(TRUE)
+ASSUME Constraint \in BOOLEAN
+
+GB ==
```

Comment:
Contextually I can see that this just determines whether GarbageCollect is ever enabled, but it would be nice to document this with a comment (same with F, Constraint, etc.)

Reply:
This is still a draft that I will use to document my steps.  There will be no need to toggle `GarbageCollect`, the constraint, fairness, ... in the final PR.